tests/test-mocha.js

```diff
@@ -1,5 +1,5 @@
-// this is the polyfil so node has a crypto.subtle etc.
-const crypto = require('isomorphic-webcrypto');
-global.crypto = crypto;
 const chai = require('chai');
 global.should = chai.should();
+
+// FIXME: setting global to avoid fixing sub-dependencies
+global.crypto = require('isomorphic-webcrypto');
```

Comment:
We'll want to fix whatever subdependencies are an issue here, but good enough to get things rolling for now.